src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let filteredMen = [];
+
+  if (century) {
+    filteredMen = people.filter(person =>
+      person.sex === 'm'
+      && (Math.ceil(person.died / 100) === century));
+  } else {
+    filteredMen = people.filter(person => person.sex === 'm');
+  }
+
+  return filteredMen.reduce((sum, person) =>
+    sum + (person.died - person.born), 0) / filteredMen.length;
```

Comment:
не делай обрыв строки
```suggestion
  return filteredMen.reduce(
    (sum, person) => sum + (person.died - person.born),
    0
  ) / filteredMen.length;
```